code/modules/holodeck/turfs.dm

```diff
@@ -154,12 +162,13 @@
 
 /turf/open/floor/holofloor/wood
 	icon_state = "wood"
+	desc = "Makes you feel at home."
 	tiled_dirt = FALSE
 
 /turf/open/floor/holofloor/snow
 	gender = PLURAL
 	name = "snow"
-	desc = "Looks cold."
+	desc = "The puffy snow clumps together to make a solid-looking floor, though still easily destroyed."
```

Comment:
considering it's a holo-floor, the description including mention of it being destructible feels wrong

Reply:
fake destructible, like when u walk on snow irl and it kinda falls under ur foot